lib/cylc/mp_pool.py

```diff
@@ -236,7 +236,7 @@ class SuiteProcPool(object):
 
     def __init__(self, size=None):
         if not size:
-            size = glbl_cfg().get(['process pool size'], size)
```

Comment:
Another instance of the same error (the only other one I found; and in this case it was of no consequence).